build.sbt

```diff
@@ -5,9 +5,7 @@ import xerial.sbt.Sonatype.sonatypeCentralHost
 
 ThisBuild / organization := "com.kevel"
 
-// Setting no cross build for the aggregating root project so that we can have proper per project exclusions.
-ThisBuild / crossScalaVersions := Nil
```

Comment:
I suppose we can move the `crossScalaVersions` from `commonSettings` to the `ThisBuild` scope here, so that we have the configuration for the Scala versions in all projects close together.

Reply:
See [e9ebdc2](https://github.com/adzerk/apso/pull/837/commits/e9ebdc2973521c1604852ae598c4bc8da5437a95).